freqtrade/exchange/__init__.py

```diff
@@ -74,21 +65,19 @@ def init(config: dict) -> None:
     # Find matching class for the given exchange name
     name = exchange_config['name']
 
-    # TODO add check for a list of supported exchanges
+    if name not in ccxt.exchanges:
+        raise OperationalException('Exchange {} is not supported'.format(name))
 
     try:
-        # exchange_class = Exchanges[name.upper()].value
         _API = getattr(ccxt, name.lower())({
             'apiKey': exchange_config.get('key'),
             'secret': exchange_config.get('secret'),
+            'password': exchange_config.get('password'),
+            'uid': exchange_config.get('uid'),
         })
-        logger.info('Using Exchange %s', name.capitalize())
```

Comment:
I think it's a useful log we shouldn't remove this

Reply:
Agreed, adding it again